src/pages/promises/index.js

```diff
@@ -128,6 +128,17 @@ export async function getStaticProps({ locale }) {
 
   const promisesApi = backend.promises();
   const promises = await promisesApi.all;
+  projectMeta.tags = Object.values(
+    promises.reduce((acc, promise) => {
+      promise.categories.forEach((category) => {
+        if (!acc[category.slug]) {
+          acc[category.slug] = category;
+        }
+      });
+
+      return acc;
+    }, {})
+  );
```

Comment:
Isn't this what is being done and returned from here https://github.com/CodeForAfrica/PromiseTracker/blob/efc0e3619bc147c8e9ab74f7e680159209cedf0e/src/lib/jsonql/promises.js#L10-L36?